sdk/sdk-common/src/orders/implementation/RefinanceParameters.ts

```diff
@@ -0,0 +1,32 @@
+import { SerializationService } from '../../services/SerializationService'
+import { IRefinanceParameters } from '../interfaces'
+import { IPosition } from '../../common/interfaces/IPosition'
+import { IPercentage } from '../../common/interfaces/IPercentage'
+
+/**
+ * @name RefinanceParameters
+ * @see IRefinanceParameters
+ */
+export class RefinanceParameters implements IRefinanceParameters {
+  readonly sourcePosition: IPosition
+  readonly targetPosition: IPosition
```

Comment:
We no longer need target position here as you refactored pool to contain just pair debt/collateral. But I can refactor this once we merge this PR.

Reply:
Changed!